.sbtopts

```diff
@@ -0,0 +1,7 @@
+-J-Xms1G
+-J-Xmx5G
+-J-XX:+UseParallelGC
+-Dfile.encoding=UTF8
+-Dscio.ignoreVersionWarning=true
+-Dorg.slf4j.simpleLogger.defaultLogLevel=info
```

Comment:
Yes, since we now dump logs into a file, we can afford to get the info level.
The terminal won't be polluted by logs and when we want to debug, we'll have similar logs from production